build/configure/dependencyGraph.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+var utils = require('utils');
+
+/**
+ * Makefile module can help you to generate new makefile, you can add new task,
```

Comment:
While you're describing a 'Makefile module', this file is named as 'dependencyGraph'. I think we should unify the name.